.github/workflows/branches.yml

```diff
@@ -37,9 +37,12 @@ jobs:
       - name: Run ${{ matrix.test-app }} test suite
         id: app_first_test_run
         env:
+          # Runs tests and builds images, but does not push (PUSH_IMAGES not set to 1)
           IMAGES_PREFIX: etnaagent/
           IS_CI: 1
           CI_SECRET: ${{ secrets.CI_SECRET }}
           IMAGES_POSTFIX: :master
+          TARGET_ROOT_BRANCH: master
```

Comment:
When comparing a branch for changes made, use master as the base for the diff.  TODO: make this relative to pull request target.